uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-assignment.js

```diff
@@ -49,6 +52,9 @@ define([
             $scope.activitySet = {};
             $scope.taskList = [];
             $scope.documentList = [];
+            $scope.activity = {
```

Comment:
The main propose of using nesting variables is to use the .(dot) notation in ng-model. Taking refrence form following  links:
- https://stackoverflow.com/questions/18128323/if-you-are-not-using-a-dot-in-your-angularjs-models-you-are-doing-it-wrong
- http://learnwebtutorials.com/why-ng-model-value-should-contain-a-dot